doc/user/olm-catalog/operator-bundles.md

```diff
@@ -0,0 +1,23 @@
+# Operator Bundles
+
+This document gives an overview of using `operator-sdk` to work with Operator [bundles][registry-bundle], namely on-disk bundle directories and creating bundle [images/metadata][registry-bundle-image].
+
+## Commands
+
+The following `operator-sdk` subcommands create or interact with Operator on-disk bundles and bundle images:
+
+* `generate csv`: creates a new or updates an existing CSV in a semantically versioned bundle directory, creates a package manifest if it does not exist, and optionally copies your CRDs to the versioned bundle directory. Read more about this command [here][sdk-generate-csv].
```

Comment:
```suggestion
* `generate csv`: creates a new or updates an existing CSV in a semantically versioned bundle directory. Also, note that it creates a `PackageManifest` if it does not exist, and optionally copies your CRDs to the versioned bundle directory. Read more about this command [here][sdk-generate-csv].
```

Reply:
"update" is closer to what `generate csv` does to on-disk files. The current version of this description doesn't go into depth about mechanics, just describes what is possible and provides a link to the full command docs. I'd prefer to keep the original description.